packages/menu/src/Menu.ts

```diff
@@ -95,6 +96,10 @@ export class Menu extends SpectrumElement {
     @query('slot:not([name])')
     public menuSlot!: HTMLSlotElement;
 
+    public override get focusElement(): HTMLElement {
+        return this.menuSlot;
```

Comment:
This breaks the focus passing into Pickers and Action Menu's, etc. (see [this VRT](https://9905080f40f3bb038d2df931c70e6c4c--spectrum-web-components.netlify.app/review/#ActionMenuStories/selects.png)). Let's chat about what the goal was for this in Office Hours tomorrow and see if we can't find a better path forward here!

Reply:
I was trying to add a focusElement getter because I was getting an error in the console, which I think you pointed out some time ago. I wasn't sure exactly how to accomplish it though 😬